src/odemis/driver/test/orsay_test.py

```diff
@@ -91,7 +93,7 @@ def setUpClass(cls):
         if TEST_NOHW == 1:
             raise unittest.SkipTest("TEST_NOHW is set. No server to contact.")
         cls.oserver = orsay.OrsayComponent(**CONFIG_ORSAY)
-
+        sleep(1)
```

Comment:
Why? If that's needed to have the code work, put it in the `__init__()`. The client shouldn't have to know that there is special timing needed.

Reply:
Riiight, my mistake. It's actually already there in __init__, so this sleep isn't needed at all. I'll delete it